app/models/katello/content_view_version.rb

```diff
@@ -231,6 +231,10 @@ def docker_images
       DockerImage.in_repositories(archived_repos).uniq
     end
 
+    def package_groups
+      archived_repos.flat_map(&:package_groups)
```

Comment:
You could probably optimize this, something like:

PackageGroup.in_repositories(archived_repos)